styles/stylesheets/screen.css

```diff
@@ -186,6 +186,23 @@ text {
   border-right: 10px solid rgba(31, 33, 36, 0.8);
 }
 
+/* line 172, ../sass/partials/_common.scss */
+.btn-filters {
+  padding: 0 5px !important;
+  margin-left: 3px;
+}
+
+/* line 177, ../sass/partials/_common.scss */
+.confirm {
```

Comment:
this is used in confirm styles/sass/partials/_common.scss as well...it;s duplicate. make it generic